migrate/20230815_add_nic_state.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  up do
+    alter_table(:nic) do
+      add_column :state, :text, null: false, default: "creating"
```

Comment:
collate C. I should add a test to check that...

 I think it's interesting that you're adding such a general `state` column, since we already have some kind of state-ish thing in Stand (but I can intuit why you can't use that one). Do you use all the states in the lifecycle you've defined?

It seems like there's a lot going on with `state`.